app-modules/ai/src/Jobs/CloneAiThread.php

```diff
@@ -78,7 +79,7 @@ public function handle(): void
             $threadReplica->user()->associate($this->recipient);
             $threadReplica->save();
 
-            foreach ($this->thread->messages as $message) {
+            foreach ($this->thread->messages()->when(SmartPromptsFeature::active(), fn ($query) => $query->whereNull('prompt_id'))->get() as $message) {
```

Comment:
I think we need to still include the smart prompt triggers when you clone a chat. Or else the history of the conversation will not be there for the AI to understand.

Reply:
Done